proto/storage/compliance_operator_v2.proto

```diff
@@ -73,31 +74,33 @@ message ComplianceOperatorRuleV2 {
 }
 
 // Next Tag: 15
-message ComplianceOperatorScanSettingV2 {
```

Comment:
For preservation, we discussed this externally.  This is not an issue:

Regarding the tables in customer environments, that isn't really a concern for 2 reasons:
I refactored the name too so if they had such a table it would be orphaned.
But most importantly for your awareness, if you pass the feature flag to the gen.go it will only apply the schema if the flag is on.
```//go:generate pg-table-bindings-wrapper --type=storage.ComplianceOperatorScanConfigurationV2 --search-category COMPLIANCE_SCAN_SETTINGS --references=storage.ComplianceOperatorProfileV2 --get-all-func --feature-flag ComplianceEnhancements```